cpp/scenario/gazebo/include/scenario/gazebo/components/WorldVelocityCmd.h

```diff
@@ -37,20 +39,29 @@ namespace ignition {
     namespace gazebo {
         // Inline bracket to help doxygen filtering.
         inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
-            struct WorldVelocity {
+            struct WorldVelocity
+            {
                 math::Vector3d linear;
                 math::Vector3d angular;
+
+                bool operator==(const WorldVelocity& other) const
+                {
+                    return this->linear == other.linear
+                           && this->angular == other.angular;
+                }
             };
 
             namespace components {
-                /// \brief A component type that contains commanded velocity of an
-                /// entity in the world frame represented by ignition::math::Vector3d.
+                /// \brief A component type that contains commanded velocity of
+                ///        an entity in the world frame represented by
+                ///        ignition::math::Vector3d.
                 using WorldVelocityCmd =
                     Component<WorldVelocity, class WorldVelocityCmdTag>;
```

Comment:
Here and in the file name `worldVelocityCmd`, what does "commanded" refer to?

Reply:
This is a data structure that is filled by methods like `Model::resetBaseVelocity`. Without studying how an ECM works in details, you can think of this data structure as a buffer between the user and the thread that implements the physics step.

If you want to know more, this data is read by the physics [here](https://github.com/robotology/gym-ignition/blob/2281c5a8bd4dea621e16eb76ca1e2e2c71fc5598/plugins/Physics/Physics.cpp#L774-L809).